src/js/view/controls/templates/menu/menu-item.ts

```diff
@@ -1,7 +1,17 @@
-import ARROW_RIGHT_ICON from 'assets/SVG/arrow-right.svg';
-export const itemTemplate = (content) => `<button type="button" class="jw-reset-text jw-settings-content-item" aria-label="${content}" dir="auto">${content}</button>`;
+import * as ARROW_RIGHT_ICON from 'assets/SVG/arrow-right.svg';
```

Comment:
have to change due to the declared module setup?

Reply:
Good call. Left in from when I was trying various fixes. Doesn't change much but I'd prefer not to make unneeded changes. Reverted!